docs/workflows/README.md

```diff
@@ -68,20 +68,27 @@ A workflow is a provider-configured set of steps that describe the process to in
 
 AWS Step functions are described in detail in the AWS documentation but they provide several benefits which are applicable to AWS.
 
-* Prebuilt solution
-* Operations Visibility
-  * Visual diagram
-  * Every execution is recorded with both inputs and output for every step.
-* Composability
-  * Allow composing AWS Lambdas and code running in other steps. Code can be run in EC2 to interface with it or even on premise if desired.
-  * Step functions allow specifying when steps run in parallel or choices between steps based on data from the previous step.
-* Flexibility
-  * Step functions are designed to be easy to build new applications and reconfigure. We're exposing that flexibility directly to the provider.
-* Reliability and Error Handling
-  * Step functions allow configuration of retries and adding handling of error conditions.
-* Described via data
-  * This makes it easy to save the step function in configuration management solutions.
-  * We can build simple interfaces on top of the flexibility provided.
+- Prebuilt solution
+- Operations Visibility
+  - Visual diagram
+  - Every execution is recorded with both inputs and output for every step.
+- Composability
+  - Allow composing AWS Lambdas and code running in other steps. Code can be run in EC2 to interface with it or even on premise if desired.
+  - Step functions allow specifying when steps run in parallel or choices between steps based on data from the previous step.
+- Flexibility
+  - Step functions are designed to be easy to build new applications and reconfigure. We're exposing that flexibility directly to the provider.
+- Reliability and Error Handling
+  - Step functions allow configuration of retries and adding handling of error conditions.
+- Described via data
+  - This makes it easy to save the step function in configuration management solutions.
+  - We can build simple interfaces on top of the flexibility provided.
+
+#### AWS Step Functions considerations
+
+- According to the
+  [AWS DeleteStateMachine documentation](https://docs.aws.amazon.com/step-functions/latest/apireference/API_DeleteStateMachine.html),
+  deleting a workflow while it has an execution in-progress will result in that
+  execution being deleted the next time it makes a state transition.
```

Comment:
Should probably include the implicit implication here. 

e.g. 

```suggestion
  execution being deleted the next time it makes a state transition.    As a result, changes to your workflow that cause Terraform to remove and re-add your workflow may result in this occurring.       If this is of concern, you should consider creating a new duplicate workflow and update related rules, then wait until the original is no longer in use.
```

That suggestion needs finessing, but hopefully gets the idea across.